src/core/graphQL-interface/GraphQL.ts

```diff
@@ -451,37 +489,48 @@ function createResolvers(core: PerspectivismCore) {
         Agent: {
             name: async (agent) => {
                 //console.debug("GQL| AGENT.name:", agent)
-                if(agent.name && agent.name !== "")
-                    return agent.name
-                else {
-                    const agentExpression = await core.languageController.getExpression(parseExprURL(agent.did))
-                    if(agentExpression)
-                        return (agentExpression.data as Agent).name
-                    else
-                        return ''
-                }
+                // if(agent.name && agent.name !== "")
+                //     return agent.name
+                // else {
+                //     const agentExpression = await core.languageController.getExpression(parseExprURL(agent.did))
+                //     if(agentExpression)
+                //         return (agentExpression.data as Agent).name
+                //     else
+                //         return ''
+                // }
```

Comment:
This code should get back in. The GraphQL client doesn't need to add these fields to the query and then this code won't get executed..